### PR TITLE
[gh-pages] update manually index to include beta release

### PR DESCRIPTION
**Pull request by paologallinaharbur**
Publishing new release currently available in fork repo only.

To test that:
```
helm repo add newrelic-beta https://newrelic-forks.github.io/helm-charts
helm upgrade --install newrelic-bundle-beta newrelic-beta/nri-bundle --devel
helm search repo newrelic --devel
```

### CODE REVIEW

File: .github/workflows/pr_index_integrity.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  gh-pages-integrity:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: check index.yaml
+        run: |
+          helm repo add test https://raw.githubusercontent.com/newrelic/helm-charts/${{ github.event.pull_request.base.ref }}
+          helm search repo newrelic -l --devel
```
Comment (roobre) [inline .github/workflows/pr_index_integrity.yaml:18]: Repo is added as `test` but tested as `newrelic`. Is this intended?

Comment (paologallinaharbur) [reply .github/workflows/pr_index_integrity.yaml:18]: is looking for all the images "newrelic"

```
NAME                             	CHART VERSION	APP VERSION	DESCRIPTION                                       
test/newrelic-infra-operator     	0.4.0        	0.5.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.3.0        	0.5.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.2.0        	0.5.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.3        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.2        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.1        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infra-operator     	0.1.0        	0.4.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.8.0        	2.9.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.3        	2.8.3      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.2        	2.8.2      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.1        	2.8.2      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.7.0        	2.8.1      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.6.4        	2.7.1      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.6.3        	2.7.0      	A Helm chart to deploy the New Relic Infrastruc...
test/newrelic-infrastructure     	2.6.2        	2.7.0      	A Helm chart to deploy the New Relic Infrastruc...
[...]
```

Comment (roobre) [reply .github/workflows/pr_index_integrity.yaml:18]: Should we somehow assert the output of `helm search repo newrelic -l --devel`? For example, if I run it on my machine:

```
helm search repo newrelic -l --devel && echo success
No results found
success
```

For example, this should fail as expected:
```shell
helm search repo newrelic -l --devel | grep -e nri-bundle | grep -- '-pre'
```

Comment (paologallinaharbur) [reply .github/workflows/pr_index_integrity.yaml:18]: oh f... I do not know why I was totally sure it was failing if nothing was returned.

I'add a grep

**Review by alvarocabanas — APPROVED** [review]
LGTM, nice addition the GH Pages Integrity

**Review by roobre — APPROVED** [review]

**Pull request merged**